namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-frontend-staging/resources/route53.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_route53_zone" "cla_frontend_staging_route53_zone" {
```

Comment:
Even though this zone is being created from the staging dir, it'll be used by all envs and is applicable to the whole hostname, so I would prefer not to see "staging" in its name, just as it's potentially confusing

Reply:
Ok shall I just use `cla-frontend` in every instance ?

Reply:
Sounds good